### PR TITLE
[8.0] Migrate hr_evaluation_interview

### PR DESCRIPTION
**Pull request by bwrsandman**
Fix critical due to null request_id
Set temporary request_id in pre migration
Set request_id to hr_evaluation.appraisal_form in post-migration

Fixes: #812 

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/hr_evaluation/migrations/8.0.0.1/pre-migration.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    for interview_id in cr.fetchall():
+        ret = user_input_obj.create(cr, SUPERUSER_ID, {
+            'survey_id': 1,
+            'type': 'link',
+        }, context={})
+        execute(
+            cr,
+            "UPDATE hr_evaluation_interview\n"
+            "SET request_id = %d WHERE id = %d\n" %
+            (ret, interview_id[0])
```
Comment (pedrobaeza) [inline addons/hr_evaluation/migrations/8.0.0.1/pre-migration.py:42]: Please avoid SQL injections here passing values as arguments.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: addons/hr_evaluation/migrations/8.0.0.1/pre-migration.py (hunk 1 above)
Comment (bwrsandman) [reply addons/hr_evaluation/migrations/8.0.0.1/pre-migration.py:42]: It has been a while... is this the proper syntax?
```
         execute(
            cr,
            "UPDATE hr_evaluation_interview\n"
            "SET request_id = %d WHERE id = %d\n",
            (ret, interview_id[0])
```

Comment (pedrobaeza) [reply addons/hr_evaluation/migrations/8.0.0.1/pre-migration.py:42]: Yeah, that's it!

**Comment by pedrobaeza** [comment]
Closing as old one.

**Pull request closed without merging**